firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -73,6 +73,10 @@
 
   /* used for thread-level synchronization of generating and persisting fids */
   private static final Object lockGenerateFid = new Object();
+
+  /* used for thread-level synchronization of getting a fid. */
```

Comment:
How about this comment:
/** Thread level synchronization for reading FID from disk. */